api/configs/feature/__init__.py

```diff
@@ -26,6 +26,10 @@ class AppExecutionConfig(BaseModel):
         description='execution timeout in seconds for app execution',
         default=1200,
     )
+    APP_MAX_ACTIVE_REQUESTS: NonNegativeInt = Field(
```

Comment:
NonNegativeInt may causes a type matching exception when env passes in an empty string. It is recommended to add a default value: 0 to `docker/docker-compose.yaml` and `docker/.env.example`.